README.md

```diff
@@ -88,6 +88,7 @@ Commands:
 Options:
   -c, [--config=<config file path>]                  # Path to the Tapioca configuration file
                                                      # Default: sorbet/tapioca/config.yml
+      [--env=key:value]                              # Environment variables to set before running Tapioca
```

Comment:
Nitpick: it might be my lack of familiarity with Thor, but it wasn't immediately obvious to me how we'd be passing multiple environment variables to the same command (I understood after reading the tests).

Could we add an example somewhere in the README of how to pass these?

Reply:
So, we already have a few `hash` type options and how they work is documented here: https://github.com/rails/thor/wiki/Method-Options#types-for-method_options

The main way is to use the `--opt=key:value key2:value2` way, but if you mark the option as `repeatable: true`, then you can also do `--opt=key:value --opt=key2:value2` too.

I guess I can add a section for how to pass env variables via command line or config file to make that more clear and link to the Thor docs for details.